modules/aggs-matrix-stats/build.gradle

```diff
@@ -16,6 +16,7 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+apply plugin: 'nebula.maven-scm'
```

Comment:
I think this is confusing to have to apply this plugin specifically. Setting hasClientJar should do this for this build.

Reply:
I don't have a strong preference. The reason I changed it to this is so that's consistent with  how we configure publishing without hasClientJar, where we only configure if the publishing plugin is applied.